tests/functional/cylc-set/00-set-succeeded/flow.cylc

```diff
@@ -0,0 +1,41 @@
+# 1. foo and bar fail incomplete.
+# 2. setter sets foo and bar to succeeded.
+# 3. foo and bar are completed, post<m> runs, scheduler shuts down.
```

Comment:
Trivial, style suggestion?
```suggestion
[meta]
    description = """
         1. foo and bar fail incomplete.
         2. setter sets foo and bar to succeeded.
         3. foo and bar are completed, post<m> runs, scheduler shuts down.
    """
```

Reply:
The `[meta]` section is really for getting documentation into the parsed workflow. For developer's eyes in small test cases, plain old comments are fine IMO.